packages/next/next-server/lib/constants.ts

```diff
@@ -12,6 +12,7 @@ export const REACT_LOADABLE_MANIFEST = 'react-loadable-manifest.json'
 export const SERVER_DIRECTORY = 'server'
 export const SERVERLESS_DIRECTORY = 'serverless'
 export const CONFIG_FILE = 'next.config.js'
+export const ENV_FILE = '.env'
```

Comment:
It's odd to me that only one variant of this file is in our constants. I digress.